nix/nixos/cardano-node-service.nix

```diff
@@ -24,7 +24,7 @@ let
             "--signing-key ${cfg.signingKey}"}"
           "${lib.optionalString (cfg.delegationCertificate != null)
             "--delegation-certificate ${cfg.delegationCertificate}"}"
-          "${cfg.extraArgs}"
+          "${toString cfg.extraArgs}"
```

Comment:
i think `] ++ cfg.extraArgs` is better, because of how `cmd` gets processed to pretty-print in the bash script

Reply:
A very good point @cleverca22.. how sloppy am I -- having introduced that mechanism in the first place..

Reply:
Fixed!